core/update.php

```diff
@@ -189,36 +189,36 @@ function update_helpful_links() {
  * Displays results of the update script with any accompanying errors.
  */
 function update_results_page() {
-  backdrop_set_title('Backdrop site update');
+  backdrop_set_title(t('Backdrop site update'));
 
   update_task_list();
   // Report end result.
   if (module_exists('dblog') && user_access('access site reports')) {
-    $log_message = 'All errors have been <a href="' . base_path() . '?q=admin/reports/dblog">logged</a>.';
+    $log_message = t('All errors have been <a href="!url">logged</a>.', array('!url' => base_path() . '?q=admin/reports/dblog'));
   }
   else {
-    $log_message = 'All errors have been logged.';
+    $log_message = t('All errors have been logged.');
   }
 
   $output = '';
   if (!isset($_SESSION['update_success'])) {
-    $output = '<p>No updates needed.</p>';
+    $output = '<p>' . t('No updates needed') . '</p>';
   }
   elseif ($_SESSION['update_success']) {
-    $output = '<p>Updates were attempted. If you see no failures below, you may proceed happily back to your <a href="' . base_path() . '">site</a>. Otherwise, you may need to update your database manually.' . ' ' . $log_message . '</p>';
+    $output = '<p>' . t('Updates were attempted. If you see no failures below, you may proceed happily back to your <a href="!url">site</a>. Otherwise, you may need to update your database manually.', array('!url' => base_path())) . ' ' . $log_message . '</p>';
   }
   else {
     $updates_remaining = reset($_SESSION['updates_remaining']);
     list($module, $version) = array_pop($updates_remaining);
-    $message = 'The update process was aborted prematurely while running <strong>update #' . $version . ' in ' . $module . '.module</strong>.' . ' ' . $log_message;
+    $message = t('The update process was aborted prematurely while running') . ' <strong>' . t('update #@version in @module.module', array('@version' => $version, '@module' => $module)) . '</strong>.' . ' ' . $log_message;
     if (module_exists('dblog')) {
-      $message .= ' ' . 'You may need to check the <code>watchdog</code> database table manually.';
+      $message .= ' ' . t('You may need to check the !watchdog database table manually.', array('!watchdog' => '<code>watchdog</code>'));
     }
     backdrop_set_message($message, 'error');
   }
 
   if (settings_get('update_free_access')) {
-    backdrop_set_message("Reminder: Don't forget to set the <code>\$settings['update_free_access']</code> value in your <code>settings.php</code> file back to <code>FALSE</code>.", 'warning');
+    backdrop_set_message(t("Reminder: Don't forget to set the !update value in your !settings file back to !false.", array('!update' => "<code>\$settings['update_free_access']</code>", '!settings' => '<code>settings.php</code>', '!false' => '<code>FALSE</code>')), 'warning');
```

Comment:
PHPCS will complain about the overlong line length for the array if it contains two or more items.
```suggestion
    backdrop_set_message(t("Reminder: Don't forget to set the !update value in your !settings file back to !false.", array(
        '!update' => "<code>\$settings['update_free_access']</code>",
        '!settings' => '<code>settings.php</code>',
        '!false' => '<code>FALSE</code>',
      )), 'warning');
```